lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -81,16 +81,7 @@ class RenderForm extends React.Component {
   }
 
   validateData(data) {
-    const errors = [],
-      regex = /^([a-zA-Z0-9_.+-])+@(([a-zA-Z0-9-])+\.)+([a-zA-Z0-9]{2,4})+$/;
-
-    if (!data.requester) {
-      errors.push(gettext('Enter a valid email address.'));
-      $('#email').closest('.form-group').addClass('has-error');
-    } else if (!regex.test(data.requester)) {
-      errors.push(gettext('Enter a valid email address.'));
-      $('#email').closest('.form-group').addClass('has-error');
-    }
```

Comment:
Why do we want to remove this?

Reply:
we are no longer taking email from user because form will be only shown to logged in user. We already have email of logged in user which dos't required validation here.